### PR TITLE
Run ansible-test sanity

### PR DESCRIPTION
**Pull request by gundalow**
Depends-On: https://github.com/ansible-network/ansible-zuul-jobs/pull/3

### CODE REVIEW

**Comment by gundalow** [comment]
Actual work is being done in https://github.com/ansible-network/ansible-zuul-jobs/pull/3

**Comment by gundalow** [comment]
retest

**Comment by pabelanger** [comment]
recheck

File: .zuul.yaml [hunk 1]
```diff
@@ -2,6 +2,4 @@
     check:
```
Comment (pabelanger) [inline .zuul.yaml:2]: if you set:

  debug: True

we should get more info from zuul

Comment (gundalow) [reply .zuul.yaml:2]: Added

**Comment by gundalow** [comment]
recheck

**Pull request closed without merging**